src/components/proofs/ProofList.tsx

```diff
@@ -28,8 +28,12 @@ const bottomWrapper = classnames(
   alignItems('items-end')
 )
 
-export function ProofListSuspended() {
-  const [category, setCategory] = useState<CategoriesTitles>('NFTs')
+export function ProofListSuspended({
+  selectedCategory,
+}: {
+  selectedCategory: CategoriesTitles
```

Comment:
I think it's better to extract this one into `interface`

Reply:
Done